lib/realtime_web/controllers/tenant_controller.ex

```diff
@@ -122,27 +121,12 @@ defmodule RealtimeWeb.TenantController do
   def delete(conn, %{"id" => id}) do
     Logger.metadata(external_id: id, project: id)
 
-    Repo.transaction(
-      fn ->
-        if Api.delete_tenant_by_external_id(id) do
-          with :ok <- UserSocket.subscribers_id(id) |> Endpoint.broadcast("disconnect", %{}),
-               :ok <- PostgresCdc.stop_all(id) do
-            :ok
-          else
-            other -> Repo.rollback(other)
-          end
-        end
-      end,
-      timeout: @stop_timeout
-    )
-    |> case do
-      {:error, reason} ->
-        Logger.error("Can't remove tenant #{inspect(reason)}")
-        send_resp(conn, 503, "")
-
-      _ ->
-        send_resp(conn, 204, "")
+    case Api.delete_tenant_by_external_id(id) do
+      true -> id |> UserSocket.subscribers_id() |> Endpoint.broadcast("disconnect", %{})
```

Comment:
tenant record can be cached and this allows users to subscribe again

Reply:
this was one of my concerns with this approach